features/FEATURE_BLE/ble/Gap.h

```diff
@@ -880,6 +1049,22 @@ class Gap {
         return BLE_ERROR_NOT_IMPLEMENTED;
     }
 
+    /**
+     * Return the type of a random address.
+     *
+     * @param[in] address The random address to retrieve the type from. The #
+     * address must be ordered in little endian.
+     *
+     * @param[out] addressType Type of the address to fill.
+     *
+     * @return BLE_ERROR_NONE in case of success or BLE_ERROR_INVALID_PARAM if
+     * the address in input was not identifiable as a random address.
+     */
+    static ble_error_t getRandomAddressType(
+        BLEProtocol::AddressBytes_t address,
+        RandomAddressType_t& addressType
```

Comment:
in user facing APIs we use pointers not refs for [out], including in this file, so for consistency this and the ones below should be pointers too